src/table/toolbar/table-toolbar.component.ts

```diff
@@ -70,13 +71,21 @@ export class TableToolbar {
 	@Input() set cancelText(value: { CANCEL: string }) {
 		this._cancelText.override(value.CANCEL);
 	}
+	@Input() showToolbarActions = true;
```

Comment:
If we keep this ...

```suggestion
	@Input() showActions = true;
```

Since we already know it's a toolbar 🙂